src/drivers/aws-dynamodb.ts

```diff
@@ -0,0 +1,161 @@
+import { createError, createRequiredError, defineDriver } from "./utils";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  GetCommand,
+  PutCommand,
+  DeleteCommand,
+  ScanCommand,
+} from "@aws-sdk/lib-dynamodb";
+
+export interface DynamoDBStorageOptions {
+  table: string;
+  region?: string;
+  credentials?: any;
+  attributes?: {
+    key?: string;
+    value?: string;
+    ttl?: string;
+  };
+  expireIn?: number;
+  client?: DynamoDBDocumentClient;
```

Comment:
Would be nice to have ttl as a top level option here, for consistenct with other drivers #262